static/sass/styles.scss

```diff
@@ -1588,3 +1588,11 @@ ol.p-list--divided .p-list__item::before {
     }
   }
 }
+
+.p-link--inverted {
```

Comment:
I added this because using the is-dark class as the docs suggest results in blue links on the meganav and AFAIK these should still be white. I'll remove if that's not the case anymore

Reply:
As far as I can see the meganav is not a child of `p-navigation`. So the `is-dark` (dark theme) of navigation has no effect on meganav.

I believe the proper fix would be to set `is-dark` on parent of the meganav (`dropdown-window`?)

<img width="1902" alt="image" src="https://github.com/canonical/ubuntu.com/assets/83575/5d19e60d-8b62-4a1a-b0ab-56d3111b7fc4">

Reply:
That seems to have done the trick!